base/inference.jl

```diff
@@ -1008,8 +1051,8 @@ function abstract_apply(af::ANY, fargs, aargtypes::Vector{Any}, vtypes::VarTable
 end
 
 function pure_eval_call(f::ANY, argtypes::ANY, atype::ANY, vtypes::VarTable, sv::InferenceState)
-    for a in drop(argtypes,1)
-        if !(isa(a,Const) || isconstType(a,false))
+    for a in drop(argtypes, 1)
+        if !(isa(a, Const) || isconstType(a, false))
```

Comment:
Would be good to take a pass through and remove these whitespace changes, since they tend to cause spurious merge conflicts. I happen to have a branch that changes these lines.